analyzer/src/main/kotlin/PackageCurationProvider.kt

```diff
@@ -21,6 +21,7 @@ package org.ossreviewtoolkit.analyzer
 
 import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.PackageCuration
+import org.ossreviewtoolkit.model.config.PackageCurationProviderConfiguration
```

Comment:
This commit should auto-close https://github.com/oss-review-toolkit/ort/issues/6055.